pom.xml

```diff
@@ -358,7 +359,20 @@
                 <scala.version>${scala_2.13.version}</scala.version>
             </properties>
         </profile>
-
+        <profile>
+            <id>spark-3.0.0</id>
+            <properties>
+                <spark.version>3.0.0</spark.version>
+                <json4s.version>3.6.6</json4s.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>spark-2.4.2</id>
```

Comment:
Why `2.4.2`? It was `2.4.4` and could maybe be bumped up. 

Same for `json4s`

Reply:
lemme check and resolve. Ideally anything 2.4.x should not be an issue , but will revert